### PR TITLE
Adding input method for Ho

### PR DESCRIPTION
**Pull request by Singkiring57**

### CODE REVIEW

File: rules/hoc/hoc-transliteration.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+			[ 'g', '𑣋' ],
+			[ 'h', '𑣙' ],
+			[ 'i', '𑣂' ],
+			[ 'j', '𑣎' ],
+			[ 'k', '𑣌' ],
+			[ 'l', '𑣚' ],
+			[ 'm', '𑣖' ],
+			[ 'n', '𑣓' ],
+			[ '𑣓g', '𑣊' ],
```
Comment (amire80) [inline rules/hoc/hoc-transliteration.js]: Doesn't work.

File: rules/hoc/hoc-transliteration.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+			[ '𑣓j', '𑣍' ],
```
Comment (amire80) [inline rules/hoc/hoc-transliteration.js]: Doesn't work.

File: rules/hoc/hoc-transliteration.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+			[ '𑣓n', '𑣐' ],
```
Comment (amire80) [inline rules/hoc/hoc-transliteration.js]: Doesn't work.

File: rules/hoc/hoc-transliteration.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+			[ 'o', '𑣉' ],
+			[ 'p', '𑣘' ],
+			[ 'q', '𑣄' ],
+			[ 'r', '𑣜' ],
+			[ '𑣜r', '𑣛' ],
```
Comment (amire80) [inline rules/hoc/hoc-transliteration.js]: Doesn't work.

File: rules/hoc/hoc-transliteration.js [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+			[ 's', '𑣞' ],
+			[ 't', '𑣒' ],
+			[ '𑣒h', '𑣕' ],
```
Comment (amire80) [inline rules/hoc/hoc-transliteration.js]: Doesn't work.

File: rules/hoc/hoc-transliteration.js [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+			[ 'u', '𑣃' ],
+			[ 'v', '𑣀' ],
+			[ 'w', '𑣟' ],
+			[ 'x', '𑣝' ],
+			[ 'y', '𑣅' ],
+
+			[ 'A', '𑢡' ],
+			[ '𑢡(i|I)', '𑢦' ],
```
Comment (amire80) [inline rules/hoc/hoc-transliteration.js]: Works with I, but not with i.

File: rules/hoc/hoc-transliteration.js [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+			[ '𑢡(U|u)', '𑢧' ],
```
Comment (amire80) [inline rules/hoc/hoc-transliteration.js]: Works with U, but not with u.

File: rules/hoc/hoc-transliteration.js [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+			[ 'B', '𑢷' ],
+			[ 'C', 'C' ],
+			[ 'C(H|h)', '𑢯' ],
```
Comment (amire80) [inline rules/hoc/hoc-transliteration.js]: Works with H, but not with h.

File: rules/hoc/hoc-transliteration.js [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+			[ 'D', '𑢱' ],
+			[ '𑢱(H|h)', '𑢴' ],
```
Comment (amire80) [inline rules/hoc/hoc-transliteration.js]: Works with H, but not h.

File: rules/hoc/hoc-transliteration.js [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+
+			[ 'G', '𑢫' ],
+			[ 'H', '𑢹' ],
+			[ 'I', '𑢢' ],
+			[ 'J', '𑢮' ],
+			[ 'K', '𑢬' ],
+			[ 'L', '𑢺' ],
+			[ 'M', '𑢶' ],
+			[ 'N', '𑢳' ],
+			[ '𑢳(G|g)', '𑢪' ],
```
Comment (amire80) [inline rules/hoc/hoc-transliteration.js]: Doesn't work.

File: rules/hoc/hoc-transliteration.js [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+			[ '𑢳(J|j)', '𑢭' ],
```
Comment (amire80) [inline rules/hoc/hoc-transliteration.js]: Doesn't work.

File: rules/hoc/hoc-transliteration.js [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+			[ '𑢳(N|n)', '𑢰' ],
```
Comment (amire80) [inline rules/hoc/hoc-transliteration.js]: Doesn't work.

File: rules/hoc/hoc-transliteration.js [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+			[ 'O', '𑢩' ],
+			[ 'P', '𑢸' ],
+			[ 'Q', '𑢤' ],
+			[ 'R', '𑢼' ],
+			[ '𑢼(R|r)', '𑢻' ],
```
Comment (amire80) [inline rules/hoc/hoc-transliteration.js]: Doesn't work.

File: rules/hoc/hoc-transliteration.js [hunk 14]
```diff
@@ -0,0 +1,102 @@
...
+			[ 'S', '𑢾' ],
+			[ 'T', '𑢲' ],
+			[ '𑢲(H|h)', '𑢵' ],
```
Comment (amire80) [inline rules/hoc/hoc-transliteration.js]: Doesn't work.

**Comment by Singkiring57** [comment]
@amire80 
I have made some changes, I hope it works this time.

**Pull request closed without merging**